requirements-dev.txt

```diff
@@ -1,7 +1,7 @@
 detect-secrets~=0.13.0
 moto~=1.3.14
 pre-commit~=1.20.0
-pytest~=5.3.1
+pytest~=4.6.7
```

Comment:
Why the downgrade?

Reply:
5.3.1 didn't seem to exist, pip told me the latest pytest was 4.6.7. But doesn't seem to be true from PyPi, so maybe something was weird on my local, I'll look

Reply:
Oh huh now it works, changing back